Makefile

```diff
@@ -59,11 +59,9 @@ services-deploy: dist/services.zip ## Upload the publish service to AWS Lambda
 	$(LOAD_ENV) \
 	&& $(SERVERLESS) deploy
 
-
 publish-service-invoke: ## Invoke the publish service
 	$(LOAD_ENV) \
-	&& $(SERVERLESS) invoke --function publish
-
+	&& curl -XPOST --fail $$PUBLISH_ENDPOINT
```

Comment:
Yep - want this variable to be evaluated by bash and not the makefile itself. So it can use the variables set in `LOAD_ENV`.

I'm not very good at Make. I think this is right?